src/exoscale/compute/api/http.clj

```diff
@@ -133,7 +133,7 @@
         (if (pos? remaining)
           ;; The previous response can be used as input
           ;; to queryAsyncJobResult directly
-          (d/chain (json-request!! config "queryAsyncJobResult" resp)
+          (d/chain (json-request!! config "queryAsyncJobResult" {:jobid jobid})
```

Comment:
as `:jobid` is part of the resp, what is the benefit from recreating parameters ?
Get ride of any other params ?

Reply:
The problem is, when the job is not ready, we have a huge job map. Then it gets merged into the main params and breaks signing the request. Probably because of nesting data.

Reply:
ok